librocksdb_sys/src/lib.rs

```diff
@@ -1127,6 +1128,17 @@ extern "C" {
     pub fn crocksdb_sstfilewriter_file_size(writer: *mut SstFileWriter) -> uint64_t;
     pub fn crocksdb_sstfilewriter_destroy(writer: *mut SstFileWriter);
 
+    // SstFileReader
+    pub fn crocksdb_sstfilereader_create(
+        path: *const u8,
+        len: size_t,
+        verify_checksum: c_uchar,
+    ) -> *mut SstFileReader;
+    pub fn crocksdb_sstfilereader_read_table_properties(
+        reader: *mut SstFileReader,
+    ) -> *mut DBTableProperties;
+    pub fn crocksdb_sstfilereader_destroy(reader: *mut SstFileReader);
```

Comment:
how can we get key-value?

Reply:
We don't need to, we can directly get the offset and the size of `seq no`, change that int the sst file.

Reply:
this is a common feature, we should support it.

Reply:
Do you mean like get an iterator for `SstFileReader`? We can't get k-v since rocksdb has no related api for `SstFileWriter`.

Reply:
what is this? https://github.com/UncP/pingcap-rocksdb/blob/uncp/sst-reader/include/rocksdb/sst_file_reader.h#L26

Reply:
I saw that before, this is a callback, we still can't get k-v. Do you mean support this callback?

Reply:
yes